flake.nix

```diff
@@ -7,7 +7,7 @@
   outputs = { self, nixpkgs, flake-utils }:
     let
       supportedSystems = {
-        "${flake-utils.lib.system.aarch64-darwin}" = "macos-x86_64";
+        "${flake-utils.lib.system.aarch64-darwin}" = "macos-aarch64";
```

Comment:
Maybe `macos-arm64`/

Reply:
You use another pair of conventions on the `nix-keysmith` side..

Reply:
There isn't an arm build for `quill` so I'm using this for both.